tests/system/feature_store/test_feature_store.py

```diff
@@ -894,6 +894,71 @@ def test_ingest_partitioned_by_key_and_time(
         orig_columns.remove("patient_id")
         assert result_columns.sort() == orig_columns.sort()
 
+    def test_passthrough_feature_set(self):
```

Comment:
As you're adding support for spark merger, we should probably add a test for that as well in `tests/system/feature_store/test_spark_engine.py`.

Reply:
done